src/Native/Unix/System.Security.Cryptography.Native.Apple/pal_trust.c

```diff
@@ -167,55 +160,102 @@ int32_t AppleCryptoNative_StoreEnumerateUserRoot(CFArrayRef* pCertsOut, int32_t*
 {
     if (pCertsOut != NULL)
         *pCertsOut = NULL;
+    if (pOSStatusOut != NULL)
+        *pOSStatusOut = noErr;
+    if (pCertsOut == NULL || pOSStatusOut == NULL)
+        return -1;
+
+    CFMutableArrayRef pCertsRef = NULL;
+    int32_t ret;
+    
+    ret = EnumerateTrust(kSecTrustSettingsDomainUser,
+                         kSecTrustSettingsResultTrustRoot,
+                         &pCertsRef,
+                         pOSStatusOut);
 
-    return EnumerateTrust(kSecTrustSettingsDomainUser,
-                          kSecTrustSettingsResultTrustRoot,
-                          pCertsOut,
-                          pOSStatusOut);
+    *pCertsOut = pCertsRef;
+
+    return ret;
 }
 
 int32_t AppleCryptoNative_StoreEnumerateMachineRoot(CFArrayRef* pCertsOut, int32_t* pOSStatusOut)
 {
     if (pCertsOut != NULL)
         *pCertsOut = NULL;
+    if (pOSStatusOut != NULL)
+        *pOSStatusOut = noErr;
+    if (pCertsOut == NULL || pOSStatusOut == NULL)
+        return -1;
 
-    CFMutableArrayRef* pCertsRef = pCertsOut;
+    CFMutableArrayRef pCertsRef = NULL;
+    int32_t ret;
 
-    int32_t ret =
-        EnumerateTrust(kSecTrustSettingsDomainAdmin, kSecTrustSettingsResultTrustRoot, pCertsRef, pOSStatusOut);
+    ret = EnumerateTrust(kSecTrustSettingsDomainAdmin,
```

Comment:
Assuming this (and similar lines) were done by `clang-format`, it's okay with me.  If not, please run `clang-format -i` on this file.

Reply:
Done.